tutorial/documents.md

```diff
@@ -0,0 +1,53 @@
+# Documents
+The Document Registry is the default mechanism for interacting with
+files in JupyterLab.  The Document Manager uses it to create models and 
+widgets for documents.  The File Browser uses the Document Manager when 
+creating and opening files.
+
+The Document Manager handles models, contexts, and widgets for documents 
+and manages their life cycle.
+
+Extensions in the JupyterLab application can register widget factories, 
+model factories, widget extensions, file types, and file creators.
+
+## Widget Factories 
+Create a widget for a given file. An example is the notebook widget 
+factory that creates NotebookPanel widgets.
+
+## Model Factories 
+Create a model for a given file.  Models are generally differentiated by 
+the contents options used to fetch the model (e.g. text, base64, notebook).
+
+## Widget Extensions
+Add additional functionality to a widget. An extension instance is created 
+for each widget.  An example is the ipywidgets extension that is created 
+for NotebookPanel widgets.  Another example would be to add a button to the 
+toolbar of each NotebookPanel widget.
+
+## File Types 
+Intended to be used in a "Create New" dialog, providing a list of known 
+file types.
+
+## File Creators 
+Intended for create quick launch file creators. The default use will be for 
+the "create new" dropdown in the file browser, giving list of items that can 
```

Comment:
"Create New" is how it's capitalized above, so whichever is the correct case should be used.